.github/workflows/test.yaml

```diff
@@ -39,8 +39,8 @@ jobs:
             - name: Unit test
               run: pnpm run unit
 
-            - name: Integration test
-              run: pnpm run integration
-
             - name: Test
               run: pnpm run test
```

Comment:
so goddamn flaky, and it runs much faster than integration test, just make sure this passes before running a longer test